libr/cons/hud.c

```diff
@@ -93,16 +93,36 @@ R_API char *r_cons_hud(RList *list, const char *prompt) {
 		r_cons_printf ("> %s|\n", buf);
 		r_list_foreach (list, iter, pos) {
 			if (!buf[0] || strmatch (pos, buf)) {
-				char *x = strchr (pos, '\t');
-				// remove \t.*
 				if (!choose || n >= choose) {
+					// remove \t.*
+					char *x = strchr (pos, '\t');
 					if (x) *x = 0;
 					p = strdup (pos);
-					for (j = 0; p[j]; j++) {
-						if (strchr (buf, p[j]))
-							p[j] = toupper ((unsigned char)p[j]);
+					if (usecolor) {
+						int last_color_change = 0;
+						char tmp;
+						r_cons_printf (" %c ", first? '-': ' ');
+						for (j = 0; p[j]; j++) {
+							if (strchr (buf, p[j])){
+								if (j > last_color_change){
+									tmp = p[j];
+									p[j]=0;
+									r_cons_printf ("%s", p + last_color_change);
+									p[j]=tmp;
+								}
+								last_color_change=j+1;
+								r_cons_printf ("%s%c\x1b[0m", Color_GREEN,  p[j]);
+							}
+						}
+						r_cons_printf ("%s\n", p + last_color_change);
+					}
```

Comment:
The else should be on the same as the brace. Please have a look here https://github.com/radare/radare2/blob/master/CONTRIBUTING.md#coding-style-guidelines for more info.